### PR TITLE
Music player Frontend Project 3344

### LINKED ISSUE #3344: Music Player Frontend Project for beginner
**Describe the project you want to add with tech stack**

Music Player Frontend Project for beginner

**Expected behavior**

One can play, Pause and many more functionality can be done there

**Screenshots (optional)**

**Additional context (optional )**

@MohitGupta121 Kindly assign me under GSSOC'23

### PR DESCRIPTION
**Pull request by soubhik-111**
# Title and Issue number 

Music player Frontend Project #3344 

Close #3344 

# Checklist:

- [X] I have mentioned the issue number in my Pull Request.
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have created a helpful and easy to understand `README.md`

### CODE REVIEW

**Comment by soubhik-111** [comment]
@MohitGupta121 
Kindly merge it or give your feedback.

**Comment by soubhik-111** [comment]
Kindly add GSSOC'23 tag

File: Frontend-Projects/Music_Player/favicon.png
Comment (MohitGupta121) [inline Frontend-Projects/Music_Player/favicon.png:1]: @soubhik-111 remove the image image from PR and upload somewhere else and use image link to load into your project.

**Review by MohitGupta121 — CHANGES_REQUESTED** [review]

File: Frontend-Projects/Music_Player/favicon.png
Comment (soubhik-111) [reply Frontend-Projects/Music_Player/favicon.png:1]: Okay, I will create a new PR.

**Pull request closed without merging**